componets/badge/badge.css

```diff
@@ -0,0 +1,52 @@
+@import url(../avatar/avatar.css);
+
+.badge__container {
+  position: relative;
+  margin: 1rem;
+}
+.badge {
+  position: absolute;
+  width: 0.93rem;
```

Comment:
try following 8 point design system. (8 point grid )
don't be too specific with decimal values.

Reply:
yes i did the change